bitrise-plugin.yml

```diff
@@ -2,10 +2,10 @@ name: init
 description: Initialize bitrise __config (bitrise.yml)__ and __secrets (.bitrise.secrets.yml)__
   based on your project.
 executable:
-  osx: https://github.com/bitrise-io/bitrise-plugins-init/releases/download/1.3.0/bitrise-plugins-init-Darwin-x86_64
-  linux: https://github.com/bitrise-io/bitrise-plugins-init/releases/download/1.3.0/bitrise-plugins-init-Linux-x86_64
+  osx: https://github.com/bitrise-io/bitrise-plugins-init/releases/download/1.3.1/bitrise-plugins-init-Darwin-x86_64
+  linux: https://github.com/bitrise-io/bitrise-plugins-init/releases/download/1.3.1/bitrise-plugins-init-Linux-x86_64
 trigger: ""
 requirements:
 - tool: bitrise
-  min_version: 1.3.0
+  min_version: 1.3.1
```

Comment:
using auto-replace you changed the min required Bitrise CLI version, I guess it was not intentional.

Reply:
🤦‍♂